infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/KubernetesTrustedCAProvisioner.java

```diff
@@ -113,26 +112,21 @@ public void provision(KubernetesEnvironment k8sEnv, RuntimeIdentity runtimeID)
     }
 
     KubernetesNamespace namespace = namespaceFactory.getOrCreate(runtimeID);
-    Optional<ConfigMap> existing = namespace.configMaps().get(configMapName);
-    if (existing.isEmpty()
-        || !(existing.get().getData() == allCaCertsConfigMap.getData()
-            || existing.get().getData().equals(allCaCertsConfigMap.getData()))) {
-      // create or renew map
-      k8sEnv
-          .getConfigMaps()
-          .put(
-              configMapName,
-              new ConfigMapBuilder()
-                  .withMetadata(
-                      new ObjectMetaBuilder()
-                          .withName(configMapName)
-                          .withAnnotations(allCaCertsConfigMap.getMetadata().getAnnotations())
-                          .withLabels(configMapLabelKeyValue)
-                          .build())
-                  .withApiVersion(allCaCertsConfigMap.getApiVersion())
-                  .withData(allCaCertsConfigMap.getData())
-                  .build());
-    }
+
+    k8sEnv
+        .getConfigMaps()
+        .put(
+            configMapName,
+            new ConfigMapBuilder()
+                .withMetadata(
+                    new ObjectMetaBuilder()
+                        .withName(configMapName)
```

Comment:
I think description of `che.infra.kubernetes.trusted_ca.dest_configmap` property should be updated

Reply:
done